index.rst

```diff
@@ -1,61 +1,643 @@
+3 <#figure-3>`__). This could be mitigated somewhat by better pixel
+flagging.
 
-.. .. rubric:: References
+.. figure:: _static/fig03.png
+   :name: figure-3
 
-.. Make in-text citations with: :cite:`bibkey`.
+   Subsections of a pre-convolved DECam image subtraction, including
+   warped and PSF-matched template, science image, and both decorrelated
+   and non-decorrelated mmatch-filtered AL difference images.
 
-.. .. bibliography:: local.bib lsstbib/books.bib lsstbib/lsst.bib lsstbib/lsst-dm.bib lsstbib/refs.bib lsstbib/refs_ads.bib
-..    :encoding: latex+latin
-..    :style: lsst_aa
+It should also be noted that currently the spatially-varying
+decorrelation (described below) is functional in the case when
+pre-convolution is also enabled. These images show the same issues as
+the non-spatially-varying version described above.
+
+**Timing:** Enabling decorrelation along with pre-convolution increases
+run-time from 55.6 to 68.0 seconds, or an increase of 22.3%. It is not
+clear why decorrelation in the case of pre-convolution increases
+run-time so significantly when it does not do so when pre-convolution is
+disabled.
+
+2. Zackay, et al. (2016) (ZOGY) image subtraction
+=================================================
+
+The Zogy algorithm is implemented in the LSST stack, and is enabled by
+setting the config ``makeDiffim.subtract='zogy'``. The main guts of the
+algorithm and its task are in the ``lsst.ip.diffim.zogy`` submodule. It
+is functional. It is implemented in pure python; although much of the
+expensive calculations are performed under-the-hood in ``C`` or
+``Fortran`` via ``scipy`` or ``afw``, be they FFTs or convolutions.
+
+We show an example Zogy diffim below in `Figure 4a <#figure-4a>`__. The
+standard Zogy implementation, in which all convolutions are performed in
+frequency space, is on the bottom-left. It shows clear signs of aliasing
+and fringing-related artifacts around bright stars. It also shows (with
+the negative artifacts near fainter stars) the effect of the apparent
+inaccurate relative flux calibration between the template and science
+images. (Note that no attempt to improve the relative calibration is
+performed in the Zogy code -- it is expected to be accurately performed
+during initial exposure calibration. This reveals a weakness of Zogy
+relative to AL -- the requirement of accurate [relative] calibration
+between the two images.) This may be seen more readily in an other
+subimage from the same DECam image (`Figure 4b <#figure-4b>`__).
+
+.. figure:: _static/fig04a.png
+   :name: figure-4a
+
+   Subsections of a DECam Zogy image subtraction, including warped and
+   PSF-matched template, science image, and the results of the
+   "standard" and image-space versions of the Zogy algorithm.
+
+.. figure:: _static/fig04b.png
+   :name: figure-4b
+
+   Subsections of the same DECam Zogy image subtraction as in Figure 4a.
+
+I should note that this fringing was observed by Tim Axelrod in another
```

Comment:
Could you give some more detail here?  What other ZOGY implementation?  On what data?

Reply:
Sorry for the omission. I include it in the document; it is this implementation: https://github.com/pmvreeswijk/ZOGY, and he was running it on DECam data. That is all I know (Tim suggested that his notes on this weren't too complete but might be worth following up with him -- apparently he figured out how to fix it by changing the way PSFEx computes its PSFs).